SimFastTiming/FastTimingCommon/src/BTLDeviceSim.cc

```diff
@@ -82,11 +85,11 @@ void BTLDeviceSim::getHitsResponse(const std::vector<std::tuple<int, uint32_t, f
         simHitAccumulator->emplace(mtd_digitizer::MTDCellId(id, row, col), mtd_digitizer::MTDCellInfo()).first;
 
     // --- Get the simHit energy and convert it from MeV to photo-electrons
-    float Npe = 1000. * hit.energyLoss() * LightYield_ * LightCollEff_ * PDE_;
+    float Npe = convertUnitsTo(0.001_MeV, hit.energyLoss()) * LightYield_ * LightCollEff_ * PDE_;
```

Comment:
@casarsa , I cannot understand what means 0.001_MeV? is it 1 keV?

Reply:
@civanch it is effectively a division by 1./1000., a convoluted way to multiply by 1000 in my opinion

Reply:
@fabiocos , I am sure that code is correct but it is not transparent, in case of mm/cm conversion there is no questions.